src/mongocxx/test/client_helpers.cpp

```diff
@@ -262,18 +271,23 @@ std::int32_t get_max_wire_version(client const& client) {
     return max_wire_version.get_int32().value;
 }
 
-std::string get_server_version(client const& client) {
-    auto output = get_server_status(client);
+std::string get_server_version() {
+    auto output = get_server_status();
     return bsoncxx::string::to_string(output.view()["version"].get_string().value);
 }
 
-document::value get_server_params(client const& client) {
-    auto reply = client["admin"].run_command(make_document(kvp("getParameter", "*")));
+document::value get_server_params() {
+    // Cache reply.
+    static auto reply = []() {
+        auto client = mongocxx::client(mongocxx::uri());
```

Comment:
```suggestion
        auto client = mongocxx::client{mongocxx::uri{}, test_util::add_test_server_api()};
```
This seems to be what is causing versioned API task failures.

Reply:
Oop. Thank you.